### PR TITLE
Adding members icon

### LINKED ISSUE #944: Add forward on member page
As kind request by the committee Stijl: add a button on the [gewis.nl/member](https://gewis.nl/member) page that forwards you to [gewis.nl/~stijl](https://gewis.nl/~stijl) (in the future [gewis.nl/stijl](https://gewis.nl/stijl)) where the files regarding the corporate design are located.

### PR DESCRIPTION
**Pull request by GEWISstijl**
Also changes the MyFuture icon used on the activities page, since I was checking whether the icons in `/img/` were acutally used. 

On merging, closes #944 

_Please note that this change requires new stylesheets to be used, so it may not work on the test branch, unless `npm run scss` is executed_

With stylish regards,
@rinkp on behalf of @GEWISstijl

### CODE REVIEW

**Comment by kokx** [comment]
Screenshot?

**Comment by rinkp** [comment]
There you go.

screenshot of members page with added button on desktop

screenshot of members page with added button on mobile

**Comment by GEWISstijl** [comment]
Hi,

Did you have a chance to look at this yet?

PS: I saw that we did not include this in the pull request, but since there has been some discussion about politics, I'll add it now: This change, together with the /corporateidentity URL, has been discussed by the board and approved in the BM of January 10th.

**Comment by tomudding** [comment]
We know everything was discussed/approved by the board. And we were working on getting this implemented (see [my fork](https://github.com/tomudding/gewisweb/commits/feature/stijl-members-page-button)), but I do not have all the time of the world to do it.

Let's start with the feedback. It is clean and does not jump out from the other buttons. However, I had the same issue with my own implementation, the GEWIS logo is not centred. I understand that none of the other icons is centred, but it is most noticeable with the GEWIS logo. I tried to fix this, but I found that there is not really a good solution. Perhaps you know something that will work?

File: module/Decision/view/decision/member/index.phtml [hunk 1]
```diff
@@ -151,6 +151,10 @@ $meetings = $this->meetings;
                             <div class="big-icon fas fa-calendar-alt"></div>
                             <div class="panel-heading"><?= $this->translate('Option calendar') ?></div>
                         </a>
+                        <a href="/corporateidentity" class="panel panel-image">
+                            <div class="big-icon gi gewis-s"></div>
+                            <div class="panel-heading"><?= $this->translate('Corporate Identity') ?></div>
+                        </a>
```
Comment (tomudding) [inline module/Decision/view/decision/member/index.phtml]: You have placed the "button" in the "Manage"-section of the Members page. This section is only visible to "active" users (i.e. users part of an organ). To my understanding, the button is supposed to be visible to all users of the website. Perhaps it should be under the "Association"-section.

**Comment by rinkp** [comment]
Awesome that you are working on it. I altered the margins in 2a4db1494da47daf2f6f76477ba6b3ca1d29acbf to make the bottom of the icons algin with the text's baseline. See the screenshots at the bottom of this message as well.

We have thought about it and adding the icon to the "Manage" section is two-fold:
1. First, it is way cleaner in the lay-out of the website
2. Second, the corporate identity will be used most by the people that are active members of GEWIS. They'll see the "manage" section, allowing them to see this button as well.

So in short: we have thought about the location, but feel absolutely free to move the button to the "Association" section, or another place that you think suits the website better. 

Please let us know if you have any questions.

screenshot van GEWIS.nl waarbij de marges van de buttons zijn aangepast (top)

screenshot van GEWIS.nl waarbij de marges van de buttons zijn aangepast (bottom)

**Comment by jszanto** [comment]
In that case, perhaps it is better to rename "Manage" to "Active members"?

**Comment by rinkp** [comment]
I've renamed the section (79c505ea9a7fb6175aa19b92e1534e4d9ec88eec) and resolved merge conflicts (a308286e188c403443af4b8566bfcf4b7991beba) for the translation files by re-applying the changes.
